sched/adaptdl_sched/allocator.py

```diff
@@ -143,7 +143,7 @@ async def _update_allocations(self, allocations):
                 LOG.info("Patch AdaptDLJob %s/%s: %s", namespace, name, patch)
                 await patch_job_status(self._objs_api, namespace, name, patch)
 
-    async def _find_nodes(self, pod_label_selector=None):
+    async def _find_nodes(self, pod_label_selector=""):
```

Comment:
I think this is subtle enough of an issue to require a comment in the code to have the default be `""`